.github/workflows/style-check.yml

```diff
@@ -40,3 +40,11 @@ jobs:
         working-directory: ./javascript
       - run: npm run lint
         working-directory: ./javascript
+  markdown-style-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: markdownlint
+          uses: nosborn/github-action-markdown-cli@v3.0.1
```

Comment:
Please specify the version with the commit id.
`nosborn/github-action-markdown-cli@018fc8b964d4d364519e7cbcda29b6b9bf3f0ba3`

Reply:
Oh, I forgot